fi/index.html

```diff
@@ -0,0 +1,85 @@
+  <meta property="og:title" content="Älä kysy kysyäksesi, vaan kysy" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
+  <meta property="og:locale" content="fi_FI" />
+  <meta property="og:image" content="https://dontasktoask.com/favicon.png"/>
+  <link rel="icon" href="/favicon.png" type="image/png" />
+  <link rel="stylesheet" type="text/css" href="/style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+</head>
+<body>
+  <main>
+    <h1>Älä kysy kysyäksesi, vaan kysy</h1>
+    <p>
+      Aina sillöin tällöin verkko keskustelu kanavilla joilla vietän aikaa, joku ilmestyy sinne ja kysyy jotain tähän suuntaan,
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Yhtään Java asiantuntijaa paikalla?
+      </p>
+    </blockquote>
+    <p>
+      Tämä on huono tapa, useista syistä. Mitä henkilö
+      <em>todella</em> haluaa kysyä on,
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Yhtään Java asiantuntijaa paikalla jotka ovat valmiita käyttämään aikaa ongelmani ratkaisemiseen,
+        mikä ikinä se onkaan, vaikka sillä ei olisikaan mitään tekemistä Javan kanssa tai jos joku
+        kuka ei tiedä mitään Javasta voisi vastata kysymykseeni?
+      </p>
+    </blockquote>
+    <p>
+      On paljon syitä miksi ihmiset kenellä ON tarvittava tieto, eivät sitä myönnä.
+      Kysymällä kysyt enemmän mitä luulet kysyväsi.
+    </p>
+    <p>
+      Pyydät ihmisiä ottamaan vastuuta ja kyseenalaistat heidän itsevarmuutta kykyihinsä.
+      Vastaan itse usein kysymyksiin jotka liittyvät kieliin tai kirjastoihin joita en ole koskaan käyttänyt.
+      Syy tähän on se että (ohjelmoijan näkökulmasta) niihin voi vastata maalaisjärjellä.
+    </p>
+    <p>
+      Vaihtoehtoisesti, sen voi nähdä näin..
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Minulla on kysymys liittyen Javaan mutta olen liian laiska oikeasti laatia sitä sanoiksi,
+        ellei kanavalla ole joku joka pystyy siihen vastaamaan.
+      </p>
+    </blockquote>
+    <p>
+      ..mikä on laiskaa. Jos et ole valmis tekemään töitä ongelmasi ratkaisemisen eteen, miksi meidän pitäisi?
+    </p>
+    <p>
+      Ratkaisu ei ole kysyä kysyäksesi, vaan kysyä. Joku joka istuu kanavalla ja vain sillöin tällöin tarkistaa mitä on meneillään,
+      tuskin vastaa "kysy kysyäksesi" kysymykseen, mutta sinun todellinen kysymyksesi saattaa herättää heidän mielenkiinnon.
+    </p>
+    <p>
+      Joten tiivistettynä, älä kysy
+      <em>"Yhtään Java asiantuntijaa paikalla?"</em>,
+      kysy mieluummin
+      <em>"Miten ratkaisen [ongelman] Javalla käyttäen [muuta kysymykseen liittyviä kirjastoja tai asioita]?"</em>
```

Comment:
> käyttäen [muuta kysymykseen liittyviä kirjastoja tai asioita]

Tuohon ainakin "muuta" tilalle "muita", mutta kuulostaa muutenkin vähän oudolta

Tämän voisi ehkä muuttaa täysin erilaiseen muotoon, esim

```suggestion
      <em>"Minulla on ongelma Javan kanssa: [tarkempi selitys]"</em>
```